find_intersecting_spherical_cones.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+def find_intersection(hashmap1, hashmap2):
+    '''Checks the condensed dictionaries hashmap1 and hashmap2 for spherical cones that both face each 
+    other and intersect.
+    
+    A new hashmap, intersecting_spheres, is returned with keys from hashmap1 and values from hashmap2 
+    '''
+    intersecting_spheres = {}
+    for sphere1, values1 in hashmap1.items(): # looping through every sphere(key)-value pair in hashmap1
+        for sphere2, values2 in hashmap2.items():
+            if face_each_other(values1, values2): # calling a function defined above
```

Comment:
Instead of saying 'calling a function defined above' provide a concise explanation of what the function is achieving? Why are we calling it here? For example, line 16 could say # check if two spherical cones face eachother